### PR TITLE
Bug 1308040 - Add "gc" schema for main pings

### PR DESCRIPTION
**Pull request by mreid-moz**
Note that per [this comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1308040#c40), the schema may be overly restrictive.

@trink can we test this against the real data to see if it's causing any significant validation failures?

### CODE REVIEW

File: templates/include/telemetry/gc.1.schema.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      "items": @TELEMETRY_GCITEM_1_JSON@
+    },
+    "random": {
+      "type": "array",
+      "maxItems": 2,
+      "items": @TELEMETRY_GCITEM_1_JSON@
+    }
+  },
+  "additionalProperties": false
+}
```
Comment (trink) [inline templates/include/telemetry/gc.1.schema.json]: Add newlines to both files

**Comment by acmiyaguchi** [comment]
[Here's a notebook](https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/8325) taking advantage of some code I wrote to figure out where all the invalid shield pings were coming from.

This should work since the current schema is currently permissive w.r.t. the gc metrics.

**Comment by trink** [comment]
I just tested against 3MM+ entries from the 3/12/2018 landfill and there were no new errors generated by the gc schema in that data set (and yes, the new schema is active)

File: templates/include/telemetry/gc.1.schema.json (hunk 1 above)
Comment (mreid-moz) [reply templates/include/telemetry/gc.1.schema.json]: done

**Comment by mreid-moz** [comment]
Thanks @acmiyaguchi and @trink, sounds like the existing data should be OK.

@trink I added the missing newlines, does this look right now?

**Pull request merged**